src/pages/iou/request/step/IOURequestStepMerchant.tsx

```diff
@@ -123,9 +97,10 @@ function IOURequestStepMerchant({
             navigateBack();
             return;
         }
-        IOU.setMoneyRequestMerchant(transactionID, newMerchant, !isEditing);
+        IOU.setMoneyRequestMerchant(transactionID, newMerchant ?? '', !isEditing);
         if (isEditing) {
             // When creating new money requests newMerchant can be blank so we fall back on PARTIAL_TRANSACTION_MERCHANT
+            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
any specific reason we want to disable this lint?

Reply:
This is a left-over, removed it, thanks for pointing it out!